timemachine/fe/single_topology.py

```diff
@@ -1725,7 +1841,7 @@ def setup_intermediate_state(self, lamb) -> VacuumSystem:
 
         return VacuumSystem(bond, angle, torsion, nonbonded, chiral_atom, chiral_bond)
 
-    def mol(self, lamb, min_bond_k=100.0) -> Chem.Mol:
+    def mol(self, lamb, min_bond_k=30.0) -> Chem.Mol:
```

Comment:
This was modified to make the test_single_topology_mol_utils.py tests pass, the initial `min_bond_k` was too high, and was set fairly arbitrarily.

Reply:
slightly different, there, `k_min=0.025`, but that's really too weak to be called anything resembling a bond.

Reply:
(note: for harmonic bonds we use `k_min = 0.1`, bug agreed that this is probably too small to be useful for visualization
https://github.com/proteneer/timemachine/blob/be34fbf458b54de5454564b67aeb1d70450a1b3f/timemachine/fe/single_topology.py#L1657)